x-pack/plugins/cases/public/components/files/files_utility_bar.test.tsx

```diff
@@ -10,16 +10,27 @@ import { screen } from '@testing-library/react';
 
 import type { AppMockRenderer } from '../../common/mock';
 import { createAppMockRenderer } from '../../common/mock';
+import { useCreateAttachments } from '../../containers/use_create_attachments';
 import userEvent from '@testing-library/user-event';
 import { FilesUtilityBar } from './files_utility_bar';
 
+jest.mock('../../containers/api');
+jest.mock('../../containers/use_create_attachments');
+jest.mock('../../common/lib/kibana');
+
+const useCreateAttachmentsMock = useCreateAttachments as jest.Mock;
```

Comment:
I was going to suggest using `jest.mocked(useCreateAttachments)` to have the mocked properties typed correctly but `UseQuery/UseMutationResult`s have a ton of props so it becomes super impractical...
It would be nice to have something like vitest's [mocked](https://vitest.dev/api/vi.html#vi-mocked), they have support for partial mocks!